src/react/useMembers.ts

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useRef, useState } from 'react';
```

Comment:
are we not adding tests for these?

Reply:
we will add for sure, I created additional ticket for this, because testing right now not that simple